### ISSUE TITLE
Error when running demo

### ISSUE THREAD

**Issue by jpdna** [ga4gh/server#1372]
I tried following instructions at:
http://ga4gh-reference-implementation.readthedocs.io/en/latest/demo.html
and am getting
```
(ga4gh-env)[jp@jp1 v1]$ ls
ga4gh-env  ga4gh-example-data  ga4gh-example-data-v4.3.tar
(ga4gh-env)[jp@jp1 v1]$ ga4gh_server 
Traceback (most recent call last):
  File "/home/jp/ga4gh_server/v1/ga4gh-env/bin/ga4gh_server", line 9, in <module>
    load_entry_point('ga4gh==0.3.3', 'console_scripts', 'ga4gh_server')()
  File "/home/jp/ga4gh_server/v1/ga4gh-env/lib/python2.7/site-packages/ga4gh/cli.py", line 145, in server_main
    parsedArgs.config_file, parsedArgs.config, parsedArgs.port)
  File "/home/jp/ga4gh_server/v1/ga4gh-env/lib/python2.7/site-packages/ga4gh/frontend.py", line 257, in configure
    dataRepository.open(datarepo.MODE_READ)
  File "/home/jp/ga4gh_server/v1/ga4gh-env/lib/python2.7/site-packages/ga4gh/datarepo.py", line 466, in open
    self.load()
  File "/home/jp/ga4gh_server/v1/ga4gh-env/lib/python2.7/site-packages/ga4gh/datarepo.py", line 1417, in load
    self._readReadGroupTable(cursor)
  File "/home/jp/ga4gh_server/v1/ga4gh-env/lib/python2.7/site-packages/ga4gh/datarepo.py", line 898, in _readReadGroupTable
    readGroup.populateFromRow(row)
  File "/home/jp/ga4gh_server/v1/ga4gh-env/lib/python2.7/site-packages/ga4gh/datamodel/reads.py", line 753, in populateFromRow
    self._sampleName = row[b'sampleName']
IndexError: No item with that key

```

Did I do something wrong?
Thanks!

**Comment by kozbo** [comment]
There was a problem with the demo data tar file. It should be fixed, see #1360

**Comment by jpdna** [comment]
after removing `ga4gh-example-data` and then extracting `ga4gh-example-data_4.4.tar`
I  get this error
```
(ga4gh-env)[jp@jp1 v1]$ ga4gh_server
Traceback (most recent call last):
  File "/home/jp/ga4gh_server/v1/ga4gh-env/bin/ga4gh_server", line 9, in <module>
    load_entry_point('ga4gh==0.3.3', 'console_scripts', 'ga4gh_server')()
  File "/home/jp/ga4gh_server/v1/ga4gh-env/lib/python2.7/site-packages/ga4gh/cli.py", line 145, in server_main
    parsedArgs.config_file, parsedArgs.config, parsedArgs.port)
  File "/home/jp/ga4gh_server/v1/ga4gh-env/lib/python2.7/site-packages/ga4gh/frontend.py", line 257, in configure
    dataRepository.open(datarepo.MODE_READ)
  File "/home/jp/ga4gh_server/v1/ga4gh-env/lib/python2.7/site-packages/ga4gh/datarepo.py", line 466, in open
    self.load()
  File "/home/jp/ga4gh_server/v1/ga4gh-env/lib/python2.7/site-packages/ga4gh/datarepo.py", line 1424, in load
    self._readRnaQuantificationSetTable(cursor)
  File "/home/jp/ga4gh_server/v1/ga4gh-env/lib/python2.7/site-packages/ga4gh/datarepo.py", line 1337, in _readRnaQuantificationSetTable
    cursor.execute("SELECT * FROM RnaQuantificationSet;")
sqlite3.OperationalError: no such table: RnaQuantificationSet
```
also didn't work for me with:
ga4gh-example-data-v4.4-alpha.tar

Could you send me the md5 of the tar file that works for you so I can double check that I am using the right one?  Thanks for your help.

**Comment by kozbo** [comment]
Clearly I am not a good QA engineer, sigh. I just reproduced your problem. I will get it fixed soon!

**Comment by kozbo** [comment]
I have uploaded a new tar file that works on my system. @jpdna Would you mind giving it a try?
https://github.com/ga4gh/server/releases/download/data/ga4gh-example-data_4.5.tar
I will leave this issue open as I still need to update the documentation.

**Comment by jpdna** [comment]
This works - thanks for the quick response @kozbo

**Comment by dcolligan** [comment]
Fixed in #1374
